Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:jessie
+
+MAINTAINER "Diego Marangoni" <diegomarangoni@me.com>
+
+RUN apt-get update
+RUN apt-get install -y curl
+RUN apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0x5a16e7281be7a449
+RUN echo deb http://dl.hhvm.com/debian jessie main | tee /etc/apt/sources.list.d/hhvm.list
```

Comment:
Why HHVM and not PHP? I am not sure sabre/katana will run correctly on HHVM yet.